src/views/Game.vue

```diff
@@ -51,30 +51,26 @@ export default {
   },
   firestore() {
     return {
-      detectives: db
+      players: db
```

Comment:
this is the main binding that allows us to read all the "players" documents. 

by default, firestore package will save an internal reference to the document id. this is what we use to update individual documents

Reply:
So now players represents both X and D's?